webpack/loaders.js

```diff
@@ -47,40 +47,28 @@ exports.html = {
   use: 'raw-loader',
 };
 
-exports.localCss = {
+exports.vendorCss = {
   test: /\.css$/,
-  include: path.resolve(process.cwd(), 'src', 'app'),
-  use: [
-    'to-string-loader',
-    'css-loader?-minimize',
-    'postcss-loader',
-  ],
-  exclude: /node_modules/,
-};
-
-exports.globalCss = {
-  test: /\.css$/,
-  include: path.resolve(process.cwd(), 'src', 'styles'),
+  include: /node_modules/,
   use: ExtractTextPlugin.extract({
     fallback: 'style-loader',
     use: [
       'css-loader?-minimize',
       'postcss-loader',
     ],
   }),
-  exclude: /node_modules/,
 };
 
-exports.svg = makeFileLoader(/\.svg$/);
-exports.eot = makeFileLoader(/\.eot$/);
-exports.woff = makeFileLoader(/\.woff$/);
-exports.woff2 = makeFileLoader(/\.woff2$/);
-exports.ttf = makeFileLoader(/\.ttf$/);
+exports.appCss = {
+  test: /\.css$/,
+  include: /src/,
```

Comment:
Thanks @smithad15, I've now included the global css to be run through the `ExtractTextPlugin` with the vendor files